src/composables/multisig.ts

```diff
@@ -71,13 +112,182 @@ export function useMultisig({ store }: UseMultisigOptions) {
+
+    let rawMultisigData: IMultisigAccount[] = [];
+
+    try {
+      await Promise.all(accounts.value.map(async ({ address }) => rawMultisigData.push(
+        ...(await fetchJson(`${activeNetwork.value.multisigBackend}/${address}`)),
+      )));
+    } catch {
+      // eslint-disable-next-line no-console
+      console.log('failed to fetch multisigAccounts');
+    }
+
+    rawMultisigData = uniqBy(rawMultisigData, 'contractId');
+
+    const result = (await Promise.all(
+      rawMultisigData.map(async ({ contractId, ...otherMultisig }) => {
+        const contractInstance = await sdk.getContractInstance({
+          source: multisigContract,
+          contractAddress: contractId,
+        });
+
+        const nonce = (await contractInstance.methods.get_nonce()).decodedResult;
+        const signers = (await contractInstance.methods.get_signers()).decodedResult;
+        const version = (await contractInstance.methods.get_version()).decodedResult;
+
+        const consensus = (
+          await contractInstance.methods.get_consensus_info()
+        ).decodedResult as any;
+
+        if (consensus.tx_hash) {
+          consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
+          // consensus.tx = TxBuilder.unpackTx((await getTransactionByHash(consensus.tx_hash)).tx);
+        }
+        consensus.expiration_height = Number(consensus.expiration_height);
+        consensus.confirmations_required = Number(consensus.confirmations_required);
+
+        return {
+          ...camelcaseKeysDeep(consensus),
+          ...otherMultisig,
+          nonce: Number(nonce),
+          signers,
+          version,
+          contractId,
+          balance: (await sdk.balance(otherMultisig.gaAccountId)) || 0,
+          address: otherMultisig.gaAccountId,
+        };
+      }),
+    ))
+      .sort((a, b) => b.balance - a.balance)
+      .sort((a, b) => b.confirmedBy.length - a.confirmedBy.length)
+      .sort((a) => a.confirmedBy
+        .some((c: string) => accounts.value.map(({ address }) => address).includes(c)) && -1);
+
+    if (!activeGaAccountId.value) {
+      setActiveGaAccountId(result[0]?.gaAccountId);
+    }
+
+    multisig.value = result;
+    storeMultisigAccounts(result, activeNetwork.value.networkId);
+  }
+
+  async function buildSpendTx(
+    senderId: string,
+    recipientId: string,
+    amount: number,
+  ) {
+    const sdk = await getSdk();
+
+    return sdk.spendTx({
+      senderId,
+      recipientId,
+      amount,
+    });
+  }
+
+  async function postSpendTx(tx: string, txHash: string) {
+    return postJson(`${activeNetwork.value.multisigBackend}/tx`, { body: { hash: txHash, tx } });
+  }
+
+  async function proposeTx(spendTx: string, contractId: string) {
+    const sdk = await getSdk();
+    const expirationHeight = (
+      (await sdk.api.getTopHeader())?.height + MULTISIG_TRANSCATION_EXPARATION_HEIGHT
+    );
+    const spendTxHash = new Uint8Array(Crypto.hash(
+      // @ts-ignore
```

Comment:
Instead of ignoring I suggest to fix the type for the `getNetworkId` - the types files says it returns a Promise, but from what I see it is not.